networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,16 +13,64 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-3"]
+== Network Observability Operator 1.3.0
+The following advisory is available for the Network Observability Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:110622[RHSA-2023:110622 Network Observability Operator 1.3.0]
+
+[id="network-observability-operator-1.3.0-features-enhancements"]
+=== New features and enhancements
+
+[id="multi-tenancy-1.3"]
+==== Multi-tenancy in Network Observability 
+* System administrators can allow and restrict individual user access, or group access, to the flows stored in Loki. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-multi-tenancy_network_observability[Multi-tenancy in Network Observability].
```

Comment:
This link is giving trouble in the Travis build - is it correct?

Reply:
Its because this is a new feature and it is not yet merged. Basically, this is an asynchronous Operator and since our releases are outside of the normal OCP timeframe, I can't merge the features until GA day. The release notes will be the last thing I merge because all the links will be broken until all the features are merged.